testsuite/elytron/modify-elytron-config.cli

```diff
@@ -0,0 +1,7 @@
+embed-server --jboss-home=${jboss.home.dir}
+/subsystem=logging/root-logger=ROOT:write-attribute(name=handlers, value=[FILE])
+/subsystem=security:remove
+/subsystem=webservices:remove
```

Comment:
It was in the original commit, I didn't use it as you can see in the subsequent commits.